internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -42,6 +42,67 @@ Resources:
         Rules:
           - ObjectOwnership: BucketOwnerEnforced
 
+  BucketPolicyForCloudFront:
```

Comment:
can't there only be one bucket policy? maybe this should just be `BucketPolicy` 🤔